lib/app.js

```diff
@@ -49,6 +49,7 @@ const MAPPINGS = {
     path: ['ds:5', 1, 2],
     fun: helper.descriptionHtmlLocalized
   },
+  shortDescription: ['ds:5', 1, 2, 73, 0, 1],
```

Comment:
Isn't this just the same as the summary below?

Reply:
Whoa. It is. Sorry, @PierfrancescoSoffritti, it looks like `summary` has what you're seeking :)